src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>"boroughs, the precise location" -&gt; "boroughs, precise location"</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>